test/Joonasw.AspNetCore.SecurityHeaders.Tests/CspBuilderTests.cs

```diff
@@ -127,6 +127,19 @@ public void RequireSriFor_ReturnsCorrectHeader()
 
             Assert.Equal("require-sri-for script", headerValue);
         }
+        
+        
+        [Fact]
+        public void RequireTrustedTypes_ReturnsCorrectHeader()
```

Comment:
A couple more unit tests would be nice. At least one with a specific policy name and one that sets up the require-trusted-types-for directive.